pippo-session-ehcache3/src/main/java/ro/pippo/session/ehcache/EhcacheSessionDataStorage.java

```diff
@@ -0,0 +1,65 @@
+/*
+ * Copyright 2016 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package ro.pippo.session.ehcache;
+
+import org.ehcache.Cache;
+import org.ehcache.CacheManager;
+import ro.pippo.session.SessionData;
+import ro.pippo.session.SessionDataStorage;
+
+/**
+ * SessionDataStorage implementation with Ehcache3.
+ * @author Herman Barrantes
```

Comment:
In general, here I prefer an empty line between the description of class and the author tag but I can live without :smile: